xla_native_functions.yaml

```diff
@@ -141,6 +141,7 @@ supported:
   - div.Tensor
   - div.Tensor_mode
   - dot
+  - einsum
```

Comment:
Seems this name or its signature needs is slightly different from the upstream reference. You can look at `csrc/generate/XLANativeFunctions.h` to find the correct method signature in your local compile call.